dialogy/plugins/text/classification/xlmr.py

```diff
@@ -33,6 +35,8 @@ class XLMRMultiClass(Plugin):
     def __init__(
         self,
         dest: Optional[str] = None,
+        timeout: float = 0.5,
+        url: str = "http://0.0.0.0:8000/",
```

Comment:
Can we rename it to something like `model_service_url`?

Reply:
Following duckling plugin conventions